### ISSUE TITLE
cDock2 and El Capitan

### ISSUE THREAD

**Issue by dljpres** [w0lfschild/cDock2#6]
Disabled SIP, uninstalled Cdock and reinstalled to cDock2, made the necessary changes under "accessibility" and yet my Transparency setting in cDock still does not work.  What am I missing?

**Comment by w0lfschild** [comment]
Duplicate of https://github.com/w0lfschild/cDock2/issues/2
